src/main/scala/TwitterStream.scala

```diff
@@ -19,6 +19,7 @@ object TwitterStream extends LazyLogging {
       .map(status => status.getText())
 
     statuses.print()
+    statuses.repartition(1).saveAsTextFiles("/user/ilos/tweets/tweets_for_timestpamp", "")
```

Comment:
please add TODO comment with a note to move path value to externalized config and change output format to CSV or JSON.